laravel-app/app/Http/Controllers/Auth/RegisterController.php

```diff
@@ -46,18 +44,24 @@ public function __construct()
         $this->middleware('guest');
     }
 
+    /**
+     * Show login form page.
+     *
+     * @return View
+     */
     public function showRegistrationForm()
```

Comment:
Since they are using Laravel 10. Laravel already removed doc block and used return type declaration.

Docs:
https://laravel.com/docs/10.x/releases#types
https://www.php.net/manual/en/language.types.declarations.php

Demo:
https://www.reddit.com/r/laravel/comments/11328e0/quick_demo_of_the_laravel_10x_shift/